identity/src/main/java/com/stripe/android/identity/injection/IdentityActivitySubcomponent.kt

```diff
@@ -5,13 +5,27 @@ import com.stripe.android.camera.CameraPermissionEnsureable
 import com.stripe.android.identity.FallbackUrlLauncher
 import com.stripe.android.identity.IdentityVerificationSheetContract
 import com.stripe.android.identity.VerificationFlowFinishable
+import com.stripe.android.identity.analytics.FPSTracker
+import com.stripe.android.identity.analytics.IdentityAnalyticsRequestFactory
+import com.stripe.android.identity.analytics.ScreenTracker
+import com.stripe.android.identity.navigation.IdentityFragmentFactory
+import com.stripe.android.identity.networking.IdentityModelFetcher
+import com.stripe.android.identity.networking.IdentityRepository
+import com.stripe.android.identity.utils.IdentityIO
 import com.stripe.android.identity.viewmodel.IdentityViewModel
 import dagger.BindsInstance
 import dagger.Subcomponent
 
 @Subcomponent
-internal interface IdentityViewModelSubcomponent {
-    val viewModel: IdentityViewModel
+internal interface IdentityActivitySubcomponent {
```

Comment:
Component naming might lead to confusion here, since we have `IdentityActivityComponent` and `IdentityActivitySubcomponent`.

Reply:
Yes `IdentityActivityComponent` was used when there is no injector available when we need to fallback recreate the graph. Renamed it to `IdentityActivityFallbackComponent`. Both of the two components is scoped to the `Activity`